WTWSMS/common/artifact_spawns/WtWSMS_artifact_spawns.txt

```diff
@@ -4,45 +4,51 @@
 
 western_regalia = {
     max_amount = 1
-    spawn_date = -1.1.1
+    spawn_date = 24.8.20
 	
 	spawn_chance = {
 		value = 100
 	}
     
     weight = {
+		# ? - 476 owned by Roman and Western Roman emperors
+		# Odoacer receives Regalia from Romulus Augustulus
+		# sent to Zeno by Odoacer, owned by Eastern Roman Emperors
+		# about 497 sent to Theodoric by Zeno
+		# further history unknown, probably lost
+		
```

Comment:
Weren't we going to transfer this to character history to avoid using the weights as workaround?

Reply:
Would it even work? Remember than artifacts are spawned on startup.

Reply:
Artifacts can be done through character history, yes.

Reply:
I don't think the current solution should be called a workaround, this is how Iron Crown of Lombardy is handled in vanilla.

Reply:
Is it? Sounds a bit convoluted.

Reply:
iron_crown_of_lombardy = {
    max_amount = 1
    spawn_date = -1.1.1
	
	spawn_chance = {
		value = 100
		modifier = {
			factor = 0
			NOT = { has_dlc = "Mystics" }
		}
	}
    
    weight = {
		value = 0
		additive_modifier = {
			value = 100
			primary_title = {
				title = k_italy
			}
			NOT = { year = 1066 }
		}
		additive_modifier = {
			value = 100
			primary_title = {
				title = e_hre
			}
			year = 1066
		}
	}
    artifacts = {
        iron_crown_of_lombardy = {
            value = 1
        }
    }
}

Reply:
If you think the solution with artifact tranferring is better and it works it can of course replace my solution, but someone would have to implement it.

Reply:
The point of it would be to avoid any oversights, but if you think this is a better solution it can remain.